app/templates/components/forms/admin/settings/billing.hbs

```diff
@@ -45,22 +45,10 @@
     <label class="required" for="adminBillingPaypalEmail">{{t 'Paypal Email'}}</label>
     <Input @type="email" @id="adminBillingPaypalEmail" @value={{this.model.adminBillingPaypalEmail}} />
   </div>
-  <form class="field">
-    <label class="required">{{t 'Country'}}</label>
-    <UiDropdown @class="search selection" @selected={{this.model.adminBillingCountry}} @forceSelection={{false}} @fullTextSearch={{true}}>
-      <Input @type="hidden" @autocomplete="no" @id="adminBillingCountry" @value={{this.model.adminBillingCountry}} />
-      <i class="dropdown icon"></i>
-      <div class="default text">{{t 'Select country'}}</div>
-      <div class="menu">
-        {{#each this.countries as |country|}}
-          <div class="item" data-value="{{country.name}}">
-            <i class="{{lowercase country.code}} flag"></i>
-            {{country.name}}
-          </div>
-        {{/each}}
-      </div>
-    </UiDropdown>
-  </form>
+  <CountryDropdown
+    @options={{this.countries}}
```

Comment:
Should automatically use countries instead of passing here

Reply:
@iamareebjamal but in someplaces they are using countries and in someplace payment countries?

Reply:
If you set prop use that, otherwise all countries

Reply:
@iamareebjamal What I think would be better is passing ```@paymentCountries={{true}}``` if paymentCountries are using as an option otherwise byDefault countries and not passing countries or paymentCountries at all, using them automatically

Reply:
OK